source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,100 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutKeys = new Set(['e', 'd', 'h', 'j', 'k', '+', '-', 'n']);
+const shortcutClass: Record<string, string> = {
```

Comment:
Should probably be a `Map`
```suggestion
const shortcutClass = new Map<string, string>([
```